client/packages/invoices/src/utils.ts

```diff
@@ -182,11 +192,20 @@ export const getStatusTranslator =
 export const isOutboundDisabled = (
   outbound: OutboundRowFragment | SupplierReturnRowFragment
 ): boolean => {
-  return (
-    outbound.status === InvoiceNodeStatus.Shipped ||
-    outbound.status === InvoiceNodeStatus.Verified ||
-    outbound.status === InvoiceNodeStatus.Delivered
-  );
+  switch (outbound.status) {
+    case InvoiceNodeStatus.New:
+    case InvoiceNodeStatus.Allocated:
+    case InvoiceNodeStatus.Picked:
+      return false;
+    case InvoiceNodeStatus.Delivered:
+    case InvoiceNodeStatus.Received:
+    case InvoiceNodeStatus.Verified:
+    case InvoiceNodeStatus.Cancelled:
+    case InvoiceNodeStatus.Shipped:
```

Comment:
Nit: Would move shipped as first case to keep it grouped with the 'Outbound statuses'